boot/aether/src/boot/aether.clj

```diff
@@ -59,13 +64,82 @@
           :password        password
           :non-proxy-hosts (get-non-proxy-hosts)}))))
 
+(defn ^{:boot/from :technomancy/leiningen} boot-home
```

Comment:
Is there already somewhere where this is available? `App` is at least reading the env variable.